nifi-commons/nifi-property-utils/src/main/java/org/apache/nifi/properties/BootstrapProperties.java

```diff
@@ -79,8 +107,53 @@ private String getPropertyKey(final String subKey) {
         return Optional.ofNullable(getProperty(getPropertyKey(BOOTSTRAP_SENSITIVE_KEY)));
     }
 
+    /**
+     * Returns the location of the HashiCorp Vault sensitive property provider
+     * @return The location of the HashiCorp Vault sensitive property provider
+     */
+    public Optional<String> getHashiCorpVaultPropertiesFile() {
```

Comment:
Do we want to introduce new methods for new property names?  Although it is convenient, and NiFi properties follows a similar approach, it seems better to maintain the generic `getProperty()` methods.  If we want to go in the direction of convenience, perhaps using an enum of known property keys is worth considering.